### ISSUE TITLE
1.0.0 relative import of bower components

### ISSUE THREAD

**Issue by akdetrick** [meetup/sassquatch2#157]
```
    >> Error: File to import not found or unreadable: ../components/julep/sass/julep.
    >>        Load path: /usr/local/git_repo/meetup-trunk
    >>         on line 11 of static/components/sassquatch2/sass/_util.scss
    >>         from line 20 of static/components/sassquatch2/sass/sassquatch.scss
    >>         from line 14 of static/style/foundation/main.scss
    >>   Use --trace for backtrace.
    Warning: Task "parallel:stage" failed. Use --force to continue.
```

Because you can't interpolate paths in `@import` sass directives, it's basically impossible to have the relative path to bower components be correct in both this repo and chapstick unless we always check in bower components in this repo.

**Comment by akdetrick** [comment]
in progress

**Comment by akdetrick** [comment]
fixed in `release/1.0.0`
